src/pages/Accounts/Accounts.module.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    
+}
+
+.tab {
+    padding-top: 24px;
+}
+
+.button {
+    width: 142px;
```

Comment:
remove the width and height, since we don't want to pass fixed values for the width/height